CHANGELOG.md

```diff
@@ -1,5 +1,43 @@
 ## 5.33.0 (Unreleased)
 
+NOTES:
+* sql: added upgrade guide to remove the deprecated `require_ssl` field from `google_sql_database_instance` resource in 6.0.0. ([#18282](https://github.com/hashicorp/terraform-provider-google/pull/18282))
+
```

Comment:
I think we can remove this - it's not a change to the provider code, and folks looking to upgrade will find it in the upgrade guide.

```suggestion
```